sentry/src/transport.rs

```diff
@@ -252,7 +251,10 @@ implement_http_transport! {
                                     disabled = Some(retry_after);
                                 }
                             }
-                            sentry_debug!("Got response: {:?}", resp.text());
+                            match resp.text() {
+                                Err(err) => { sentry_debug!("Failed to read sentry response: {}", err); },
+                                Ok(text) => { sentry_debug!("Get response: `{}`", text); },
```

Comment:
```suggestion
                                Err(err) => sentry_debug!("Failed to read sentry response: {}", err),
                                Ok(text) => sentry_debug!("Get response: `{}`", text),
```

Reply:
I'm wondering how this does not fail rustfmt.

Reply:
its inside the macro, so none of that stuff is rustfmt-ed, also, I needed to wrap this in a block because of the way the `sentry_debug` macro is built :-(

Reply:
I see, thanks. At least the comment after the `}` should go usually.